browser-checklist.md

```diff
@@ -6,9 +6,9 @@
 - [ ] IE11
 - [ ] IE10
 - [ ] IE9
-- [ ] IE8
+- [ ] IE8 - No Javascript
```

Comment:
This might be slightly vague, but making it more explicit could get long. Perhaps use an asterisk and put a note at the bottom of the list like, "JavaScript support is not required, but essential content should be readable."